public/app/core/navigation/kiosk.ts

```diff
@@ -11,6 +11,8 @@ export function getKioskMode(queryParams: UrlQueryMap): KioskMode | null {
     case '1':
     case true:
       return KioskMode.Full;
+    case 'embed':
```

Comment:
Not sure using kiosk mode to add the embed option is the best idea here, since we need the dashboard embed view to be on a separate URL anyways, but this at least minimises addition of extra code.